Week9Serialization/Week9Serialization/BinaryFormatter.cs

```diff
@@ -0,0 +1,84 @@
+﻿using KellermanSoftware.CompareNetObjects;
+using System;
+using System.IO;
+using System.Runtime.Serialization;
+using System.Runtime.Serialization.Formatters.Binary;
+
+namespace Week9Serialization
+{
+    /// <summary>
+    /// Uses BinaryFormatter to serialize, deserialize, and compare data.
+    /// </summary>
+    public class BinaryFormat : ISerializer
+    {
+        /// <summary>
+        /// Uses ISerializer interface to serialize data using BinaryFormatter.
+        /// </summary>
+        /// <param name="list"></param>
+        static void SerializeList(VehicleList<Vehicle> list)
```

Comment:
Static methods cannot implement interface methods. This was one of your build errors.